include/fiction/algorithms/network_transformation/inverter_balancing.hpp

```diff
@@ -0,0 +1,320 @@
+//
+// Created by benjamin on 13.04.23.
+//
+
+#ifndef FICTION_INVERTER_BALANCING_HPP
+#define FICTION_INVERTER_BALANCING_HPP
+
+#include "fiction/traits.hpp"
+#include "fiction/types.hpp"
+#include "fiction/utils/name_utils.hpp"
+
+#include <mockturtle/traits.hpp>
+#include <mockturtle/utils/node_map.hpp>
+#include <mockturtle/views/fanout_view.hpp>
+#include <mockturtle/views/topo_view.hpp>
+
+#include <type_traits>
```

Comment:
It seems that some includes are missing. I spotted `<vector>`, `<cstdint>`, `<algorithm>`